examples/cloudml-bee-health-detection/trainer/task.py

```diff
@@ -52,58 +52,52 @@ def _initialise_params():
     args_parser.add_argument(
         '--training_path',
         help='Location to training data.',
-        default='gs://mybucket/salt_segmentation/data/csv/train.csv'
+        default='gs://bee-health/data/train.csv'
     )
     args_parser.add_argument(
         '--validation_path',
         help='Location to validation data.',
-        default='gs://mybucket/salt_segmentation/data/csv/valid.csv'
+        default='gs://bee-health/data/valid.csv'
     )
     args_parser.add_argument(
         '--image_path',
         help='Location of image folder.',
-        default='gs://mybucket/salt_segmentation/train/images/'
+        default='gs://bee-health/bee_imgs/'
     )
     args_parser.add_argument(
-        '--depth_mean',
-        help='Mean depth on training data.',
-        required=True,
-        type=float
-    )
-    args_parser.add_argument(
-        '--depth_std',
-        help='Standard deviation of depth on training data.',
-        required=True,
-        type=float
+        '--subspecies_embedding',
+        help='Size of subspecies embedding.',
+        default=1,
```

Comment:
Why default to 1 column if there are 4 subspecies?

Reply:
thats the default size for each embedding not the number of subspecies. This will learn 4 embeddings of size 1 for each subspecies.

The default size 1 came from the hyper parameter tuning job. It was part of the best set of hyper-parameters,

Reply:
Weird.  Do you have any intuition to back up why the hyperparameter tuner would choose 1 column?  I'd have expected 3-4, like if the data was dummy encoded or 1-hot encoded.

Reply:
yes, we only have a handful of values for that column and I think the model can encode the important differences between them  (for this task) on a single number (e.g. the value of this number is enough to encode the info the model needs to decide how to evaluate the picture differently)